helm.tf

```diff
@@ -99,191 +82,830 @@ resource "helm_release" "metrics_server" {
 
 }
 
+resource "helm_release" "cert_manager" {
+  count      = var.helm_cert_manager_enabled || var.k8s_opentelemetry_enabled ? 1 : 0
+  name       = "cert-manager"
+  namespace  = "cert-manager"
+  repository = "https://charts.jetstack.io"
+  chart      = "cert-manager"
+  version    = "1.6.1"
+
+  set {
+    name  = "installCRDs"
+    value = true
+  }
 
-resource "helm_release" "loki_stack" {
-  count             = var.helm_loki_stack_enabled ? 1 : 0
-  name              = "loki-stack"
+  depends_on = [time_sleep.wait_20_seconds]
+
+}
+
+# ========================= monitoring charts ========================= #
+
+resource "helm_release" "prometheus_stack" {
+  count             = var.helm_prometheus_enabled ? 1 : 0
+  name              = "prometheus-stack"
+  namespace         = "monitoring"
+  create_namespace  = true
+  repository        = "https://prometheus-community.github.io/helm-charts"
+  chart             = "kube-prometheus-stack"
+  version           = "35.0.3"
```

Comment:
We should allow chart version selection (module input)

Reply:
I'm marking this as resolved because we agreed to don't do it